docs/docs/telemetry.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Telemetry
+---
+
+Gatsby contains a telemetry feature that collects anonymous usage information that is used to help improve Gatsby for all users.
+The Gatsby user base is growing very rapidly. It's important that our small team and the greater community will better understand the usage patterns, so we can best decide how to design future features and prioritize current work.
+
+You will be notified when installing Gatsby and when running it for the first time.
+
+## How to opt-out
+
+Users may always opt-out from the telemetry with `gatsby telemetry --disable` or setting the environment variable `GATSBY_TELEMETRY_DISABLED` to `1`
+
+## Why?
+
+**Anonymous** aggregate user analytics allow us to prioritize fixes and features based on how and when people use Gatsby.
+Since much of Gatsby’s function revolves around community plugins and starters, we want to collect information on usage
+and reliability so that we can ensure a high-quality ecosystem.
+
+This raises a question. How will we use this telemetry data to improve the ecosystem? Some examples are helpful:
```

Comment:
All these sentences need changed to present tense as when this PR is merged, things will be live.

Reply:
or when we finally take these steps.

Reply:
I think it's safe to change it to present tense now. As soon as data starts coming in, we're going to be using it for many of these ideas